third_party/terraform/website/docs/d/datasource_google_compute_backend_service.html.markdown

```diff
@@ -21,7 +21,7 @@ data "google_compute_backend_service" "baz" {
 
 resource "google_compute_backend_service" "default" {
   name          = "backend-service"
-  health_checks = ["${tolist(data.google_compute_backend_service.baz.health_checks)[0]}"]
+  health_checks = [tolist(data.google_compute_backend_service.baz.health_checks)[0]]
```

Comment:
I feel like there is a better 0.12 of doing this but I'm not remembering off the top of my head.

Reply:
This is one of two that I did by hand, because `terraform 0.12upgrade` ran into a syntax error.  I changed it to this and it seemed to be fine, but that doesn't mean there isn't a better way.  I can ask someone :)

Reply:
Sounds like, because `health_checks` is a set, and not a list, that's why we need `tolist`.  And from the documentation, only one health check is currently allowed, which is how we can get away with using the 0th index and get what we want.  The suggestion was to make it a list, but that's out of scope for this, and the consensus was that this is still legal `0.12` syntax.